deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -57,7 +57,35 @@ spec:
       {{- end }}
       {{- if $options.hasConfigMap }}
       shareProcessNamespace: true
+      {{- end }}
       initContainers:
+      - image: {{ include "nvidia-device-plugin.fullimage" . }}
+        name: gpu-feature-discovery-imex-init
+        command:
+        - "/bin/bash"
+        - "-c"
+        - |
+          if [[ -f /config/etc/nvidia-imex/nodes_config.cfg ]]; then
+            echo "Removing cached IMEX nodes config"
+            rm -f /config/etc/nvidia-imex/nodes_config.cfg
+          fi
+
+          if [[ ! -f /driver-root/etc/nvidia-imex/nodes_config.cfg ]]; then
+            echo "No IMEX nodes config path detected; Skipping"
+            exit 0
+          fi
+
+          echo "Copying IMEX nodes config"
+          mkdir -p /config/etc/nvidia-imex
+          cp /driver-root/etc/nvidia-imex/nodes_config.cfg /config/etc/nvidia-imex/nodes_config.cfg
```

Comment:
Can we make `/etc/nvidia-imex/nodes_config.cfg` a local envvar to this script instead of repeating it everywhere.

Reply:
Done in latest.